doozer/doozerlib/backend/konflux_olm_bundler.py

```diff
@@ -684,9 +684,10 @@ async def _update_konflux_db(
             source_repo = df.labels['io.openshift.build.source-location']
             commitish = df.labels['io.openshift.build.commit.id']
 
+            component_name = df.labels['com.redhat.component']
             version = df.labels['version']
             release = df.labels['release']
-            nvr = "-".join([metadata.get_olm_bundle_short_name(), version, release])
+            nvr = "-".join([component_name, version, release])
```

Comment:
Let's keep metadata.get_olm_bundle_short_name() for bundles. component_name should be a mistake.

Reply:
I mean this is for Konflux bundle build and the `com.redhat.component` label only matters for Brew builds. Before the PR of removing the -container suffix, we didn't have -container suffix either.
But if you have a reason to add -container suffix for konflux bundles, I am fine. This has no functional impact to FBC build.

Reply:
I'd say let's leave the -container suffix for consistency with image builds and Brew legacy naming